requirements.txt

```diff
@@ -0,0 +1,10 @@
+certifi==2025.6.15
```

Comment:
This might be a separate discussion, so I can split the requirements.txt out into its own PR if desired.